.github/workflows/validate.yml

```
@@ -0,0 +1,47 @@
+name: Validate newly added JSON
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  validate-json:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          # Needed to push changes back to the repo
+          fetch-depth: 0
```

This workflow doesn't push, does it?
```suggestion
          persist-credentials: false
```